vendor/github.com/burntsushi/toml/.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
```

Comment:
It looks to me like `go mod` doesn't do pruning of `.travis`, `.gitignore`, `README` and other non-go files.  Is there any value in adding a post-processing step?  I'm thinking it might be best to just live with it.

Reply:
I went ahead and added a little hacky delete to the revendor make target. It doesn't catch _everything_ that `dep` did, but it's a start and we can add or remove things from it as necessary going forward.